README.md

```diff
@@ -268,6 +268,6 @@ We welcome your contributions! Please have a read of
 [chat]: https://badges.gitter.im/webpack/webpack.svg
 [chat-url]: https://gitter.im/webpack/webpack
 
-[dev-ware]: /webpack/webpack-dev-middleware
-[hot-client]: /webpack-contrib/webpack-hot-client
+[dev-ware]: https://github.com/webpack/webpack-dev-middleware#options
+[hot-client]: https://github.com/webpack/webpack-dev-middleware#options
```

Comment:
not sure why you changed this one

Reply:
Really? I open these links on GitHub and they are https://github.com/webpack-contrib/webpack-serve/blob/master/webpack/webpack-dev-middleware and https://github.com/webpack-contrib/webpack-serve/blob/master/webpack-contrib/webpack-hot-client.

Reply:
no I mean you changed hot-client's link to point to webpack-dev-middleware. Line 272 doesn't look like you made the change you were going for there

Reply:
Sorry about that, fixed.